### DISCUSSION TITLE
Sub-crates

### DISCUSSION THREAD

**Discussion by s1nth** [WebCrateApp/webcrate#49] [Ideas]
It would be really cool to be able to create subfolders.

**Comment by BetaHuhn** [comment]
This has already been requested, see [#28](https://github.com/WebCrateApp/webcrate/discussions/28).

I am currently not planning on adding sub-crates/nested crates any time soon. I will keep it in mind though will probably come back to it at a later point.

As mentioned in https://github.com/WebCrateApp/webcrate/discussions/28#discussioncomment-1253227, tags are something that might help you in the meantime. They will be available in one of the next versions. Let me know if that would work for you as well.
